frontend/src/pages/question/questionBuilder/questionTemplate/QuestionTemplate.jsx

```diff
@@ -15,7 +16,8 @@ function extractDetails(questionTemplate) {
   const newTypes = [];
   const newLabels = [];
   const newCuries = [];
-  questionTemplate.query_graph.nodes.forEach((node) => {
+  Object.keys(questionTemplate.query_graph.nodes).forEach((nodeId) => {
```

Comment:
https://user-images.githubusercontent.com/1634866/105094869-e59d6080-5a72-11eb-8feb-5d0855976b61.mov

Clearing the selection in a template leads to a change from chemical_substance -> biolink:ChemicalSubstance.